### DISCUSSION TITLE
Grief Prevention help

### DISCUSSION THREAD

**Discussion by xipsaaahoy** [GriefPrevention/GriefPrevention#1223] [General]
I have a question, I have the griefprevention.ignoreclaims permissions and when I try to ignore a claim it says I dont have the permisisons to ignore basic claims!

**Comment by bigpresh** [comment]
What permissions management plugin are you using?

If you added the permission via config files, did you remember to use whatever command your permissions management system uses to reload changes?

What GriefPrevention version are you using, and on what Minecraft version?
